demo/components/date-picker-demo.tsx

```diff
@@ -3,6 +3,7 @@ import { DatePicker, DatePickerProps } from '../../src';
 
 export interface DatePickerDemoState {
     value: Date;
+    startingDay: number;
 }
 
 export class DatePickerDemo extends React.Component<Partial<DatePickerProps>, DatePickerDemoState> {
```

Comment:
I think demos should be a bit more appealing, with some sense of real world use.
I suggest flight date picker for depart and return dates...